sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClientBuilder.java

```diff
@@ -68,7 +69,7 @@ public OkHttpAsyncHttpClientBuilder addNetworkInterceptor(Interceptor networkInt
      * @param networkInterceptors the interceptors to add
      * @return the updated OkHttpAsyncHttpClientBuilder object
      */
-    public OkHttpAsyncHttpClientBuilder setNetworkInterceptors(List<Interceptor> networkInterceptors) {
+    public OkHttpAsyncHttpClientBuilder networkInterceptors(List<Interceptor> networkInterceptors) {
         this.networkInterceptors = Objects.requireNonNull(networkInterceptors, "networkInterceptors cannot be null.");
```

Comment:
Should this check for both null and empty?

Reply:
Only checking nullness should suffice, no `Interceptors` is allowed.